server/server.js

```diff
@@ -1,28 +1,18 @@
-const express = require('express');
-const app = express();
-const morgan = require('morgan');
-const path = require('path');
-const cors = require('cors');
-const port = process.env.PORT || 3002;
-const db = require('../database/db');
-const dbUtils = require('../database/dbUtils');
-const Models = require('../database/models')
+const mongoose = require('mongoose');
+const app = require('./app.js');
 
-app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, '../public')));
-app.use('/:id', express.static(path.join(__dirname, '../public')));
-app.use(cors());
+mongoose.connect((process.env.MONGODB_URI || 'mongodb://localhost/galleries'), { useCreateIndex: true, useNewUrlParser: true });
```

Comment:
db.js is using `localhost/gallery`.  is this file not being used anymore? if so remove `db.js`